packages/end-to-end/cypress/support/commands.js

```diff
@@ -1,25 +1,246 @@
+  cy.intercept('GET', 'https://maps.googleapis.com/maps-api-v3/api/js/54/11/marker.js').as(
+    'markerJsRequest',
+  );
+
+  cy.get('[data-cy=navbar-option]')
+    .eq(1)
+    .should('exist')
+    .and('not.be.disabled')
+    .click({ force: true });
+
+  //arrive at farm map page and draw a field
+  cy.url().should('include', '/map');
+
+  cy.get('[data-cy=spotlight-next]', { timeout: 60 * 1000 })
+    .should('exist')
+    .and('not.be.disabled')
+    .click();
+  cy.get('[data-cy=spotlight-next]').should('exist').and('not.be.disabled').click();
+  cy.get('[data-cy=spotlight-next]').should('exist').and('not.be.disabled').click();
+  cy.get('[data-cy=map-addFeature]').should('exist').and('not.be.disabled').click();
+
+  cy.wait('@googleMapsApiCall');
+  cy.wait('@markerJsRequest');
+  cy.waitForReact();
+
+  // Select "Field"
+  cy.contains(fieldString).click();
+
+  cy.get('[data-cy="map-selection"]').should('be.visible');
+  cy.get('[data-cy=mapTutorial-continue]').should('exist').and('not.be.disabled').click();
+
+  cy.get('[data-cy=map-mapContainer]').click(500, 300);
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(400, { log: false });
+  cy.get('[data-cy=map-mapContainer]').click(700, 300);
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(400, { log: false });
+  cy.get('[data-cy=map-mapContainer]').click(700, 400);
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(400, { log: false });
+  cy.get('[data-cy=map-mapContainer]').click(500, 400);
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(400, { log: false });
+  cy.get('[data-cy=map-mapContainer]').click(500, 300);
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(400, { log: false });
+  cy.get('[data-cy=mapTutorial-continue]').should('exist').and('not.be.disabled').click();
+  cy.get('[data-cy=map-drawCompleteContinue]').should('exist').and('not.be.disabled').click();
+
+  cy.get('[data-cy=areaDetails-name]').should('exist').type('First Field');
+  cy.get('[data-cy=createField-save]').should('exist').and('not.be.disabled').click();
+
+  cy.get('[data-cy=snackBar').should('exist').and('be.visible');
+  cy.get('div[data-cy="snackBar"]').find('[class*="button"]:first').click();
+
+  cy.waitForReact();
+
+  // Confirm that location exists
+  cy.visit('/');
+  cy.get('[data-cy=home-farmButton]').should('exist').and('not.be.disabled').click({ force: true });
+  cy.get('[data-cy=navbar-option]')
+    .eq(1)
+    .should('exist')
+    .and('not.be.disabled')
+    .click({ force: true });
+  cy.contains('First Field').should('be.visible');
+
+  // Check that it is in Redux
+  cy.window()
+    .its('store')
+    .invoke('getState')
+    .its('entitiesReducer')
+    .its('gardenReducer')
+    .its('ids')
+    .should('not.be.empty');
+});
+
+Cypress.Commands.add('acceptSlideMenuSpotlights', (crop_menu_name) => {
```

Comment:
These last three commands, `addFarm`, `onboardCompleteQuestions` and `acceptSlideMenuSpotlights` are not reused in other test files but just by other commands here. With that in mind, they could probably just be regular functions instead of commands https://docs.cypress.io/api/cypress-api/custom-commands#1-Dont-make-everything-a-custom-command